DependencyInjection/ActionConfiguration/ListActionConfiguration.php

```diff
@@ -38,6 +38,17 @@ protected function buildParametersTree(NodeParentInterface $node)
                     ->end()
                 ->end()
             ->end()
+            ->arrayNode('sorting')
+                ->canBeEnabled()
+                ->children()
+                    ->scalarNode('sort_by')
```

Comment:
Yep, better than nothing :)

... The real point is : I'm not really comfortable with configuration, and it was sufficient for my needs 😳